src/Files.App/Views/PaneHolderPage.xaml

```diff
@@ -122,7 +122,7 @@
 				<icore:ChangePropertyAction
 					PropertyName="CurrentInstanceBorderBrush"
 					TargetObject="{Binding ElementName=PaneLeft}"
-					Value="{ThemeResource ControlStrokeColorDefault}" />
+					Value="{ThemeResource DividerStrokeColorDefaultBrush}" />
```

Comment:
Is this correct? I think we need a custom brush so that we have the correct color in each theme.

Reply:
I've confirmed that brush is appropriate in light(light dark), dark(light dark), high contrast(white). But previous one was red in hc theme. but we probably need to wrap this brush like App.Theme.DividerStrokeColorDefaultBrush.

Reply:
> but we probably need to wrap this brush like App.Theme.DividerStrokeColorDefaultBrush.

That was my assumption